tests/behaviour/contracts/builtins/block.sol

```diff
@@ -0,0 +1,17 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+pragma solidity >=0.5.0;
+
+
+contract WARP {
+
+    function number() external view returns (bool) {
+        uint256 y = block.number;
+        return y > 0;
+    }
+ 
+    // Testnet returns 0 for get_block_timestamp so there is no test.
```

Comment:
If you only have this file up, this comment is confusing. I'd suggest spelling it out that there is no associated expectation and that it is here only to verify that the function can compile into compilable cairo